dev/com.ibm.ws.org.apache.cxf.ws.security/bnd.bnd

```diff
@@ -1,41 +0,0 @@
-#*******************************************************************************
```

Comment:
Did you delete the entire bundle project or just the bnd.bnd? Only the deleted bnd.bnd is showing in the review page, so if you haven't already make sure you delete the entire project.